MaggieKimani1/STACKOVERFLOWLITE-V1
# #162570889 User can post a question

What does this PR do?
Enables user to post a question

Description of Task to be completed?
Creating an API for user to post a question

How should this be manually tested?
Clone this repository
https://github.com/MaggieKimani1/STACKOVERFLOW-LITE
Run on postman

What are the relevant pivotal tracker stories?
What does this PR do?
Builds a login interface page for Users

Description of Task to be completed?
Create a login form
Style page using css

How should this be manually tested?
Clone this repository
https://github.com/MaggieKimani1/STACKOVERFLOW-LITE
Open login.html in a browser

What are the relevant pivotal tracker stories?
#162570889
